src/Api/Controllers/NotificationController.cs

```diff
@@ -34,5 +36,16 @@ public IActionResult GetNotifications()
             else
                 return NotFound();
         }
+
+        // POST api/Notifications/:id/markread
+        [HttpPost("{id}")]        
+        public IActionResult Post(int userId)
+        {
+            var result = notificationManager.UpdateIsReadOrNot(userId);
```

Comment:
Update**IsReadOrNot** .. her i come ...
Use a unambiguous name like MarkAs Read, and no, it's not the userId.